### PR TITLE
changed domains and database layer for on_home_page

### PR DESCRIPTION
**Pull request by mythicat554**

### CODE REVIEW

**Comment by ahsu1230** [comment]
@tonywu315 can you give this a quick review? For now, they're only adding a column to the repo layer (controllers, integrations, stuff will come in a future PR).

@mythicat554 can you let Tony know what you're trying to accomplish? You can link the Asana task here

**Comment by mythicat554** [comment]
hi @tonywu315 , expanding on what Aaron said, I'm just trying to add a boolean "on_home_page" to the announcements database. here's the asana task: https://app.asana.com/0/0/1172147013853485/f
I would really appreciate it if you could help me out, thanks!

File: constellations/orion/src/tests_integration/announce_integration_test.go [hunk 1]
```diff
@@ -40,13 +40,15 @@ func Test_CreateAnnouncements(t *testing.T) {
 	assert.EqualValues(t, 3, announces[0].Id)
 	assert.EqualValues(t, "Author 3", announces[0].Author)
 	assert.EqualValues(t, "Message 3", announces[0].Message)
+	assert.EqualValues(t, false, announces[0].OnHomePage)
 	assert.EqualValues(t, 2, announces[1].Id)
 	assert.EqualValues(t, "Author 2", announces[1].Author)
 	assert.EqualValues(t, "Message 2", announces[1].Message)
-	assert.EqualValues(t, 3, len(announces))
```
Comment (tonywu315) [inline constellations/orion/src/tests_integration/announce_integration_test.go:46]: Keep this line at the end

File: constellations/orion/src/repos/migrations/000002_create_table_announce.up.sql [hunk 2]
```diff
@@ -7,6 +7,7 @@ CREATE TABLE announcements
     posted_at  datetime     NOT NULL,
     author     varchar(255) NOT NULL,
     message    text         NOT NULL,
+    on_home_page  boolean,
```
Comment (tonywu315) [inline constellations/orion/src/repos/migrations/000002_create_table_announce.up.sql:10]: Make sure the default value is false

**Pull request merged**